libraries/azure_mysql_database.rb

```diff
@@ -0,0 +1,45 @@
+require 'azure_generic_resource'
+
+class AzureMySqlDatabase < AzureGenericResource
+  name 'azure_mysql_database'
+  desc 'Verifies settings for an Azure MySQL Database'
+  example <<-EXAMPLE
+    describe azure_mysql_database(resource_group: 'rg-1', server_name: 'mysql-server-1' name: 'customer-db') do
+      it { should exist }
+    end
+  EXAMPLE
+
+  def initialize(opts = {})
+    # Options should be Hash type. Otherwise Ruby will raise an error when we try to access the keys.
+    raise ArgumentError, 'Parameters must be provided in an Hash object.' unless opts.is_a?(Hash)
+
+    opts[:required_parameters] = %i(server_name)
+    opts[:resource_path] = [opts[:server_name], 'databases'].join('/')
+    opts[:resource_provider] = specific_resource_constraint('Microsoft.DBforMySQL/servers', opts)
+    opts[:resource_identifiers] = %i(database_name)
+
+    # static_resource parameter must be true for setting the scene in the backend.
```

Comment:
`for setting the resource_provider` ?

Reply:
this is the same across all static resource AFAICT

Reply:
Yes, it is the same for all. There are some additional checks as well, that's why `scene` is used. But we can replace or remove it if it is adding confusion.

Reply:
Fixed.